### PR TITLE
feat(angular): migrate versions to 14.2.0

### PR DESCRIPTION
**Pull request by Coly010**
Adds support for Angular 14.2.0.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,7 +76,7 @@
     "@rollup/plugin-image": "^2.1.0",
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-node-resolve": "^13.0.4",
-    "@schematics/angular": "~14.1.0",
+    "@schematics/angular": "~14.2.0",
     "@storybook/addon-essentials": "~6.5.9",
     "@storybook/addon-knobs": "~6.3.0",
```
Comment (leosvelperez) [inline package.json:81]: `ng-packagr` version needs to be updated.

File: packages/angular/migrations.json [hunk 2]
```diff
@@ -1469,6 +1469,46 @@
         "jest-preset-angular": {
           "version": "~12.2.0",
           "alwaysAddToPackageJson": false
+        },
+        "@angular-devkit/architect": {
```
Comment (leosvelperez) [inline packages/angular/migrations.json:1473]: We need to add a migration generator to bump `@angular/cli`.

**Review by leosvelperez — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Coly010) [reply package.json:81]: Good spot

File: packages/angular/migrations.json (hunk 2 above)
Comment (Coly010) [reply packages/angular/migrations.json:1473]: A migration generator? So we can’t do it in this file anymore?

**Review by leosvelperez — APPROVED** [review]
LGTM! 🎉

**Pull request merged**